File: src/sdk-reference/cpp/1/auth/get-my-rights/snippets/get-my-rights.cpp
try {
  kuzzle->auth->login("local", R"({"username":"foo","password":"bar"})");

  std::vector<std::shared_ptr<kuzzleio::UserRight>> rights =
    kuzzle->auth->getMyRights();

  for (auto right : rights) {
    std::cout << right->controller() << " " << right->action() << std::endl;
    std::cout << right->index() << " " << right->collection() << std::endl;
    std::cout << right->value() << std::endl;
  }
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/auth/get-my-rights/snippets/get-my-rights.cs
try {
  kuzzle.auth.login("local", @"{""username"":""foo"",""password"":""bar""}");

  List<UserRight> rights =
    kuzzle.auth.getMyRights();

  foreach (var right in rights) {
    Console.WriteLine(right.controller() + " " + right.action());
    Console.WriteLine(right.index() + " " + right.collection());
    Console.WriteLine(right.value());
  }
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.getMessage());
}
